src/test/scala/com/exasol/cloudetl/bucket/S3BucketTest.scala

```diff
@@ -77,26 +77,35 @@ class S3BucketTest extends AbstractBucketTest {
     assertS3Bucket(bucket, configMappings - "fs.s3a.session.token")
   }
 
+  private[this] def bucketWithDefaultConnectionString(properties: Map[String, String]): Bucket = {
+    val identifier = "S3_ACCESS_KEY=access;S3_SECRET_KEY=secret;S3_SESSION_TOKEN=token"
+    val exaMetadata = mockConnectionInfo("", identifier)
+    getBucket(properties, exaMetadata)
+  }
+
   test("apply returns S3Bucket with access, secret and session token from connection") {
     properties = defaultProperties ++ Map(
       "CONNECTION_NAME" -> "connection_info"
     )
-    val exaMetadata =
-      mockConnectionInfo("", "S3_ACCESS_KEY=access;S3_SECRET_KEY=secret;S3_SESSION_TOKEN=token")
-    val bucket = getBucket(properties, exaMetadata)
-    assertS3Bucket(bucket, configMappings)
+    assertS3Bucket(bucketWithDefaultConnectionString(properties), configMappings)
```

Comment:
I would rename this method, what does it do? I can't say by the name

Reply:
Fixed.